enterprise/backend/src/metabase_enterprise/data_editing/api.clj

```diff
@@ -474,6 +475,16 @@
                                      :action_id   action_id})))]
       (data-editing.describe/describe-unified-action unified scope input))))
 
+(def configure
+  "A temporary var for our proxy in [[metabase.actions.api]] to call, until we move this endpoint there."
+  (api.macros/defendpoint :post "/configure"
+    ""
```

Comment:
Let's at least put a basic docstring here:

```suggestion
    "This API returns a data representation of the form the FE will render. It does not update the configuration."
```

Since nothing is using this yet, maybe we should actually rename the API to `/configure-form`...

Reply:
doesn't it more grammatically correct to call it "configuration-form"?